targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/TARGET_MUSCA_A1_NS/device/cmsis_nvic.h

```diff
@@ -0,0 +1,33 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 Arm Limited
```

Comment:
Can you add SPDX identifiers to new files please

Reply:
So all the target files should have SPDX identifiers? (100+ files)

Reply:
Yes

Reply:
Done